templates/custom/nodes/node--outdoor-location--teaser.html.twig

```diff
@@ -84,7 +84,7 @@
 
```

Comment:
Isn't this the teaser that doesn't have alt attributes too?

Reply:
Not in this PR. :)